src/frontend/src/handler/util.rs

```diff
@@ -191,6 +195,55 @@ impl CompatibleSourceSchema {
     }
 }
 
+pub fn gen_query_from_table_name(from_name: ObjectName) -> crate::error::Result<Query> {
+    let table_factor = TableFactor::Table {
+        name: from_name,
+        alias: None,
+        for_system_time_as_of_proctime: false,
+    };
+    let from = vec![TableWithJoins {
+        relation: table_factor,
+        joins: vec![],
+    }];
+    let select = Select {
+        from,
+        projection: vec![SelectItem::Wildcard(None)],
+        ..Default::default()
+    };
+    let body = SetExpr::Select(Box::new(select));
+    Ok(Query {
+        with: None,
+        body,
+        order_by: vec![],
+        limit: None,
+        offset: None,
+        fetch: None,
+    })
+}
+
+pub fn gen_query_from_logstore_ge_rw_timestamp(
+    logstore_name: ObjectName,
+    rw_timestamp: i64,
+) -> crate::error::Result<Statement> {
+    let sql_str = format!(
+        "SELECT * FROM {} WHERE kv_log_store_epoch >= {} ORDER BY kv_log_store_epoch",
+        logstore_name, rw_timestamp
+    );
+    let query_stmt = Parser::parse_sql(&sql_str)
+        .map_err(|err| ErrorCode::InternalError(format!("Parse fetch to select error: {}", err)))?
+        .pop()
+        .ok_or_else(|| ErrorCode::InternalError("Can't get fetch statement".to_string()))?;
+    Ok(query_stmt)
+}
+
+pub fn convert_epoch_to_logstore_i64(epoch: u64) -> i64 {
+    epoch as i64 ^ (1i64 << 63)
+}
+
+pub fn convert_logstore_i64_to_epoch(logstore_i64: i64) -> u64 {
+    logstore_i64 as u64 ^ (1u64 << 63)
+}
```

Comment:
BTW, the log store using i64 to represent epoch seems hard to use, we could select a negative number for kv_log_store_epoch, cc @wenym1 

```
  kv_log_store_epoch  | kv_log_store_seq_id | kv_log_store_vnode | kv_log_store_row_op |  v1  |  v2  |  v3  |     t1._row_id
----------------------+---------------------+--------------------+---------------------+------+------+------+--------------------
 -9217204332329369600 |                   0 |                 50 |                   1 |  999 |  999 |  999 | 394733093828894720
```